boards/mulle/Makefile.include

```diff
@@ -0,0 +1,261 @@
+export CXX = $(PREFIX)g++
+export AR = $(PREFIX)ar
+export AS = $(PREFIX)as
+export LINK = $(PREFIX)gcc
+export SIZE = $(PREFIX)size
+export OBJCOPY = $(PREFIX)objcopy
+export TERMPROG ?= $(RIOTBASE)/dist/tools/pyterm/pyterm
+export FLASHER ?= $(OPENOCD)
+export DEBUGGER ?= $(GDB)
+export DEBUGSERVER ?= $(OPENOCD)
+export RESET ?= $(OPENOCD)
+
+# define build specific options
+CPU_USAGE = -mcpu=cortex-m4
+FPU_USAGE = -mfloat-abi=soft -msoft-float
+export CFLAGS += \
+    $(DEBUG_CFLAGS) \
+    -std=gnu99 \
+    $(OPTIMIZATION) \
+    -Wall -Wstrict-prototypes -Werror=implicit-function-declaration \
+    $(CPU_USAGE) \
+    $(FPU_USAGE) \
+    -mlittle-endian \
+    -mthumb \
+    -fno-common \
+    -fshort-enums \
+    -ffunction-sections \
+    -fdata-sections \
+    -fno-builtin \
+    -fno-strict-aliasing \
+    -fsigned-char \
+    #
+export CXXFLAGS += \
+    -ffunction-sections \
+    -fdata-sections \
+    -fno-builtin \
+    #
+export ASFLAGS += \
+    $(DEBUG_CFLAGS) \
+    $(CPU_USAGE) \
+    $(FPU_USAGE) \
+    -mlittle-endian \
+    #
+export LINKFLAGS += \
+    $(DEBUG_CFLAGS) \
+    -std=gnu99 \
+    $(CPU_USAGE) \
+    $(FPU_USAGE) \
+    -mlittle-endian \
+    -static \
+    -mthumb \
+    -nostartfiles \
+    -Wl,--fatal-warnings \
+    #
+export LINKFLAGS += -T$(LINKERSCRIPT)
+export OFLAGS ?= -O binary
+
+ifdef BUILD_WITH_CLANG
+ifneq ($(BUILD_WITH_CLANG),0)
+export CFLAGS += -target $(TARGET_TRIPLE)
+export CXXFLAGS += -target $(TARGET_TRIPLE)
+export LINKFLAGS += -target $(TARGET_TRIPLE)
+export CC       = clang
+export CXX      = clang++
+export LINK     = clang
+export LLVMPREFIX ?= llvm-
+export AS       = $(LLVMPREFIX)as
+export AR       = $(LLVMPREFIX)ar
+export NM       = $(LLVMPREFIX)nm
+# There is no LLVM linker yet, use binutils.
+#export LINKER   = $(LLVMPREFIX)ld
+# objcopy does not have a clear substitute in LLVM
+#export OBJCOPY  = $(LLVMPREFIX)objcopy
+export OBJDUMP  = $(LLVMPREFIX)objdump
+# LLVM lacks a binutils strip tool as well...
+#export STRIP    = $(LLVMPREFIX)strip
+export SIZE     = $(LLVMPREFIX)size
+
+# Since Clang is not installed as a separate instance for each crossdev target
+# we need to tell it where to look for platform specific includes (Newlib
+# headers instead of Linux/Glibc headers.)
+# On GCC this is done when building the cross compiler toolchain so we do not
+# actually need to specify the include paths for system includes.
+# Ubuntu gcc-arm-embedded toolchain (https://launchpad.net/gcc-arm-embedded)
+# places newlib headers in several places, but the primary source seem to be
+# /etc/alternatives/gcc-arm-none-eabi-include
+# Gentoo crossdev places newlib headers in /usr/arm-none-eabi/include
+# Ubuntu also seem to put a copy of the newlib headers in the same place as
+# Gentoo crossdev, but we prefer to look at /etc/alternatives first.
+NEWLIB_INCLUDES ?= \
+  -isystem /etc/alternatives/gcc-$(TARGET_TRIPLE)-include \
+  -isystem /usr/$(TARGET_TRIPLE)/include \
```

Comment:
for OS X this could be `/usr/local/opt/$(TARGET_TRIPLE)*/$(TARGET_TRIPLE)/include`